apps/src/gamelab/AnimationPicker/AnimationPicker.jsx

```diff
@@ -1,4 +1,4 @@
-var actions = require('./animationPickerModule');
+import * as actions from './animationPickerModule';
 var AnimationPickerBody = require('./AnimationPickerBody.jsx');
```

Comment:
AirBnb style guide says this is a no-no. https://github.com/airbnb/javascript#modules--no-wildcard I'm not sure I agree with it, but mabye we should make a note in our style guide if we're going to allow it?